libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -457,6 +466,27 @@ public void onActivityResult(int requestCode, int resultCode, @Nullable Intent d
         return otherMediaOptions;
     }
 
+    private ArrayList<MediaOption> initOtherMediaFileOptions() {
+        ArrayList<MediaOption> otherMediaOptions = new ArrayList<>();
+
+        Bundle arguments = getArguments();
+        FragmentActivity activity = getActivity();
+        if (activity == null || arguments == null) {
+            AppLog.e(T.EDITOR,
+                    "Failed to initialize other media options because the activity or getArguments() is null");
+            return otherMediaOptions;
+        }
+
+        String packageName = activity.getApplication().getPackageName();
+
+        int stockMediaResourceId =
```

Comment:
What do you think about renaming this to `
```suggestion
        int chooseFileResourceId =
```
for more clarity?

Reply:
Yes, it was bad copy paste :) Thanks.